### PR TITLE
feature - search screen UI changes

### PR DESCRIPTION
**Pull request by sjacobs1**
##Description
Search Screen UI Changes :
rounded images
Better spacing
display release year and rating

Search functionality implemented

##GIF
ezgif com-optimize-4

##Checklist
- [x]  The code follows the project's coding standards.
- [ ]  100% Unit test coverage for Tested View Models
- [x]  All warnings Resolved.

### CODE REVIEW

File: MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewController.swift [hunk 1]
```diff
@@ -20,7 +21,26 @@ class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITable
     override func viewDidLoad() {
         super.viewDidLoad()
         setupTableView()
-        searchMoviesViewModel.fetchSearchedMovies()
+        setupSearchBar()
+        noResultsLabel.isHidden = true
+    }
+
+    private func setupSearchBar() {
+        searchMovie.delegate = self
```
Comment (sjacobs1) [inline MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewController.swift]: public funcs then private. fix

File: MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewController.swift [hunk 2]
```diff
@@ -7,11 +7,12 @@
 
 import UIKit
 
-class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate {
 
     // MARK: - IBOutlets
     @IBOutlet private weak var searchMovieTableView: UITableView!
     @IBOutlet private weak var searchMovie: UISearchBar!
+    @IBOutlet weak var noResultsLabel: UILabel!
```
Comment (sjacobs1) [inline MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewController.swift:16]: private

**Review by ksookdhew — APPROVED** [review]

File: MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewModel.swift [hunk 3]
```diff
@@ -9,6 +9,8 @@ import Foundation
 
 protocol ViewModelDelegate: AnyObject {
```
Comment (kieran-woodrow) [inline MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewModel.swift:10]: please call this ViewModelProtocol

**Review by kieran-woodrow — APPROVED** [review]

File: MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewModel.swift (hunk 3 above)
Comment (sjacobs1) [reply MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMoviesViewModel.swift:10]: will address

**Pull request merged**